packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -518,6 +523,7 @@ export class SceneTreeController {
           const req = new FilterRequest();
           req.setFilter(term);
           req.setFullTree((options.includeCollapsed ?? true) === true);
+          req.setExactMatch((options.exactMatch ?? false) === true);
```

Comment:
Can't this just be represented as `req.setExactMatch(!!options.exactMatch)`?

Also, the above line can be simplified to the opposite:
`req.setFullTree(!options.includeCollapsed)`

Reply:
Good idea. Updated.